server/webapp/WEB-INF/rails/webpack/helpers/spark_routes.ts

```diff
@@ -70,6 +70,10 @@ export default class {
     }
   }
 
+  static showDashboardPath(): string {
+    return "/go/api/dashboard";
```

Comment:
@varshavaradarajan -- This should accept a view name and pass it along as a query param.

Reply:
Oh okay. Will fix it. Sorry, didn't check with the PR run.